lib/big_query/timestamp_rollups/daily.ex

```diff
@@ -24,10 +24,10 @@ defmodule BigQuery.TimestampRollups.Daily do
   end
 
   def range(from, to) do
-    range(floor(from), floor(to), [])
+    range(floor(from), ceiling(to), [])
```

Comment:
All of these rollups changed somewhat ... mostly to standardize how they did ranges.  Check the tests to actually see how they behave (the range array no longer includes the `?to=` date).